### ISSUE TITLE
Allow resource newrelic_cloud_aws_integrations to enable all services

### ISSUE THREAD

**Issue by heavinjay** [newrelic/terraform-provider-newrelic#2258]
### Feature Description
The resource newrelic_cloud_aws_integrations currently only supports enabling six services however there are other services that the AWS metric stream method does not support some outlined at the bottom of this document https://docs.newrelic.com/docs/infrastructure/amazon-integrations/connect/aws-metric-stream/
For those services to collect the data we need to use the API Polling account connection and then enable these services. One of such is S3. If you enable additional services outside of this resource then an error is thrown I'm guessing because this resource cannot manage the additional ones that were enabled. The error you get is Error: maximum retries reached: ERR_FORBIDDEN
  on new_relic.tf line 202, in resource "newrelic_cloud_aws_integrations" "enable_services":
 202: resource "newrelic_cloud_aws_integrations" "enable_services" {

Currently there is no way to use this resource if you wish to enable additional services outside of the 6 supported.

### Describe Alternatives
Currently the only alternative is to use Terraform to create the account link but then to leave out the newrelic_cloud_aws_integrations resource and instead manually enable the services that you need.

**Comment by nedl86** [comment]
@kidk FYI this appears to be very similar to #2253 which I raised last week.

**Comment by muhd-aslm** [comment]
Yes, people who want to employ the polling approach really require this. It would be preferable if terraform could be used to integrate polling for all AWS services that are available via newrelic UI.

**Comment by Nandu-pns** [comment]
Hi Everyone - @heavinjay, @nedl86, @muhd-aslm 

The latest terraform provider version (v3.24.0) now provisions the functionality to automate all the available AWS polling integrations via terraform. Please checkout our latest [docs](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/resources/cloud_aws_integrations) for the information on the same. Thank you !
